### ISSUE TITLE
Accessibility menu is not accessible via xpath.

### ISSUE THREAD

**Issue by sowmikshovon** [sowmikshovon/wppool_SQA#4]
When disabling keyboard shortcut from accessibility menu the “Accessibility” option is not clickable(From my limited observation) when uisng xpath. Maybe this menu is accessible from suggestions from other settings pages or directly through a link.
